apps/coscrad-frontend/src/coscrad-theme-provider.tsx

```diff
@@ -30,6 +30,7 @@ export const CoscradThemeProvider = ({ children }: CoscradThemeProviderProps): J
                 },
             },
             typography: {
+                fontFamily: ['"Helvetica"', 'Roboto', 'Arial', 'sans-serif'].join(','),
```

Comment:
I take it that you need the Helvetica to be wrapped in double quotes and the others unquoted? Just trying to reason about why there's an extra set of quotes on that one.